packages/subgraph/tests/token/token-voting.test.ts

```diff
@@ -1,57 +1,23 @@
-import {
-  assert,
-  clearStore,
-  log,
-  logStore,
-  test
-} from 'matchstick-as/assembly/index';
-import {Address, bigInt, BigInt, Bytes} from '@graphprotocol/graph-ts';
+import {assert, clearStore, test} from 'matchstick-as/assembly/index';
+import {bigInt, BigInt} from '@graphprotocol/graph-ts';
 
 import {
   handleVoteCast,
   handleProposalExecuted,
   handleVotingSettingsUpdated,
   _handleProposalCreated
 } from '../../src/packages/token/token-voting';
-import {TokenVotingPlugin} from '../../generated/schema';
-import {VOTER_OPTIONS, VOTING_MODES} from '../../src/utils/constants';
+import {VOTING_MODES} from '../../src/utils/constants';
 import {
-  ADDRESS_ONE,
   DAO_TOKEN_ADDRESS,
-  CONTRACT_ADDRESS,
   STRING_DATA,
-  DAO_ADDRESS,
-  PROPOSAL_ID,
   VOTING_MODE,
```

Comment:
Is this big import removal (with no code affected) because of the builders being introduced earlier?

Reply:
yes, i forgot to remove them in the previous pr.